lib/screens/publish_screen.dart

```diff
@@ -591,99 +562,54 @@ class _OwnerBottomDrawerState extends State<OwnerBottomDrawer> {
     await provider.repository.launchMyUrl(url: provider.nft.url.changeDomain());
   }
 
-  Widget buildRow({required String title, required String subtitle}) {
+  Widget buildRow({required String title, required String subtitle, final viewIPFS = false, final bool canCopy = false}) {
     final viewModel = context.watch<EaselProvider>();
+
     return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
       children: [
-        Expanded(
-          child: Text(
-            title,
-            style: _rowTitleTextStyle,
-          ),
+        Text(
+          title,
+          style: _rowTitleTextStyle.copyWith(fontWeight: FontWeight.w500),
         ),
-        Expanded(
-            child: subtitle.length > 14
-                ? Row(
-                    children: [
-                      Text(
-                        subtitle.substring(0, 8),
-                        style: _rowTitleTextStyle,
-                      ),
-                      const Text("...",
-                          style: TextStyle(
-                            color: Colors.white,
-                          )),
-                      Text(
-                        subtitle.substring(subtitle.length - 5, subtitle.length),
-                        style: _rowTitleTextStyle,
-                      ),
-                      SizedBox(
-                        width: 1.w,
-                      ),
-                      clipboardWidget(subtitle)
-                    ],
-                  )
-                : Row(
-                    children: [
-                      InkWell(
-                        onTap: () {
-                          if (title == "asset_uri".tr()) {
-                            onViewOnIPFSPressed(
-                              provider: viewModel,
-                            );
-                          }
-                        },
-                        child: Text(
-                          subtitle,
-                          style: title == "asset_uri".tr() ? _rowTitleTextStyle.copyWith(color: EaselAppTheme.kLightPurple) : _rowTitleTextStyle,
-                        ),
-                      ),
-                      SizedBox(
-                        width: 1.w,
-                      ),
-                      if (title == "content_identifier".tr()) clipboardWidget(subtitle)
-                    ],
-                  ))
+        Row(
+          mainAxisAlignment: MainAxisAlignment.end,
+          children: [
+            InkWell(
```

Comment:
Why view ipfs is there when its not even clicable?

Reply:
because it is common widget for both Copy cid and view on IPFS